COMMUNICATION.md

```diff
@@ -1,51 +1,62 @@
-# Communicating within the iOS team
+# Communicating with your colleagues
 
-## Why we have guidelines about communicating
-Within our iOS team we mostly communicate with each other via the magic of the internet, and that is great! It's easy and very efficient. But online communication can easily result into misunderstandings, which tend to be long, time consuming and inefficient. That's why we created some guidelines for how we like to communicate professionally, keep them in the back of your head when commenting on GitHub or Slack.
+Within our iOS team, we mostly communicate with each other via the magic of the internet, and that's great! It's easy and very efficient, especially since we're often in different cities and/or countries. 
 
-## When to pay attention to the guidelines
-Obviously we're communicating all the time. We can't read this document every time we're having something come out of our mouths or keyboards. That would result in silencing the world 🤐, and thats the opposite of the intention of these guidelines.
+But online communication can easily result into misunderstandings, which tend to be long, time consuming and inefficient.These guidelines are meant to solve issues in our professional communication, like pull request comments, code review and more thorough discussions.
 
-- **Only in professional communication**
-When making jokes, talking about your favourite tea, or planning a team members birthday party surely a lot of things can go wrong in communication as well. But these guidelines are meant to solve issues in our professional communication, like pull request comments, code review and more thorough discussions.
+Keep them in the back of your head when commenting on GitHub or Slack.
 
-- **When writing short replies**
-When you write a reply of less than ~5 words, have an alarm bell in the back of your head 🔔. Is it just tacit agreement? Then it’s probably fine! “Great job”, “Couldn’t have put it better”,  “toats awesome!” are all fine. In every other case though: 🚨!
+## Table of contents
 
-It good practice to elaborate. It may seem inefficient to be so verbose in our replies, but it will avoid misunderstandings, which can result in way greater inefficiency.
+- [Short replies](#short-replies)
+- [Criticizing or asking questions](#criticizing-or-asking-questions)
+- [Giving advice or tips](#giving-advice-or-tips)
+- [Clarifications and disagreements](#clarifications-and-disagreements)
+
+## Short replies
+
+When you write a reply of less than ~5 words, have an alarm bell in the back of your head 🔔. Is it just tacit agreement? Then it’s probably fine! “Great job”, “Couldn’t have put it better”,  “totes awesome!” are all fine. 
+
+In every other case though: 🚨! It's good practice to elaborate. It may seem inefficient to be so verbose in our replies, but it will avoid misunderstandings, which can result in far greater inefficiency.
+
+## Criticizing or asking questions
```

Comment:
👀 what happened here?

Reply:
🇺🇸 spelling to be internally consistent with everything I was rewriting